src/components/TabAbout/Contact/index.jsx

```diff
@@ -249,26 +267,48 @@ function Contact() {
                                         <h4 className="d-flex align-items-center float-left">
                                             <i className="feather-home me-2"></i>
                                             {userInfo.city + ", " + userInfo.district}</h4>
-                                        <i onClick={() => setCityStatus(!cityStatus)}
-                                           className="ti-pencil d-flex font-md float-right cursor-pointer hover-edit"></i>
+                                        {userInfo.id === currentUser.id ?
+                                            <i onClick={() => setCityStatus(!cityStatus)}
+                                               className="ti-pencil d-flex font-md float-right cursor-pointer hover-edit">
+                                            </i>
+                                            : null
```

Comment:
Should return fragment tag <></> instead of null value

Reply:
Resolved!